src/main/java/spireMapOverhaul/zones/candyland/consumables/rare/Banana.java

```diff
@@ -0,0 +1,29 @@
+package spireMapOverhaul.zones.candyland.consumables.rare;
+
+import com.megacrit.cardcrawl.characters.AbstractPlayer;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import spireMapOverhaul.zones.candyland.actions.changeMaxHPAction;
+import spireMapOverhaul.zones.candyland.consumables.AbstractConsumable;
+
+import static spireMapOverhaul.SpireAnniversary6Mod.makeID;
+import static spireMapOverhaul.util.Wiz.atb;
+
+
+public class Banana extends AbstractConsumable {
+    public final static String ID = makeID(Banana.class.getSimpleName());
+
+    public Banana() {
+        super(ID, 0, CardType.SKILL, CardRarity.RARE, CardTarget.SELF);
+        baseMagicNumber = magicNumber = 2;
```

Comment:
Discussing the rarity stuff reminded me -- you should add `tags.add(CardTags.HEALING);` to all the cards that give a permanent benefit (the four rares).